packages/main/src/ResponsivePopover.hbs

```diff
@@ -9,7 +9,22 @@
 		@ui5-afterClose="{{_afterDialogClose}}"
 	>
 		{{!-- forward slot header to inner dialog slot header --}}
-		<slot slot="header" name="header"></slot>
+		{{#if header.length}}
+			<slot slot="header" name="header"></slot>
+		{{else}}
+			<header class="ui5-responsive-popover-header">
+				{{#if headerText}}
+					<ui5-title class="ui5-responsive-popover-header-text">{{headerText}}</ui5-title>
```

Comment:
the header text is smaller in openui5 you have to double check the required font-size